### ISSUE TITLE
Organic revival surgery is doable on synths

### ISSUE THREAD

**Issue by nikothedude** [NovaSector/NovaSector#2459]
Reporting client version: 515.1635

## Issue Summary

Title. The surgery requires no bodytype.

## Round ID:
[10826](https://scrubby.melonmesa.com/round/10826)

## Testmerges:
- [[PORT] [SEMI-MODULAR] [JANK] Ports the lovely Character Directory which was ported from Coyote Bayou](https://github.com/NovaSector/NovaSector/pull/183)
- [Ooey Gooey Slimegirl Buffs Vol. 1](https://github.com/NovaSector/NovaSector/pull/1661)
- [[SEMI-MODULAR] Taur rework Pt. 1: Serpentine taurs](https://github.com/NovaSector/NovaSector/pull/1953)
- [Ouroboros rejigging](https://github.com/NovaSector/NovaSector/pull/2088)
- [[Semi-Modular] Ports over additional Crusher varients from monkestation](https://github.com/NovaSector/NovaSector/pull/2238)
- [Neuters synthetic electrical damage ](https://github.com/NovaSector/NovaSector/pull/2300)
- [[S] Ruin spawn tweak](https://github.com/NovaSector/NovaSector/pull/2357)
- [Entombed + paraplegic quirk interaction & minor QoL update](https://github.com/NovaSector/NovaSector/pull/2367)
- [[MODULAR] double-weapon rebalance](https://github.com/NovaSector/NovaSector/pull/2433)
- [mold blaster - mold structures are now mostly blastable](https://github.com/NovaSector/NovaSector/pull/2438)

## Reproduction:

Just try starting organic revival surgery on a synth.

**Comment by nikothedude** [comment]
id fix this, but im not sure its the best change, as it would make revival surgery impossible on organics with robotic chests

**Comment by Floofies** [comment]
I agree, we should fix #2769 alongside this to avoid a further decrease in what's already a low QOL.

**Comment by Floofies** [comment]
Fixed by #4503
